maat-court-data-api/src/main/java/gov/uk/courtdata/courtdataadapter/client/CourtDataAdapterClient.java

```diff
@@ -53,7 +53,8 @@ public void postLaaStatus(LaaStatusUpdate laaStatusUpdate, Map<String,String> he
                         .body(BodyInserters.fromValue(laaStatusUpdateJson))
                         .retrieve();
 
-        log.info("LAA status update posted {}", Optional.of( clientResponse.toBodilessEntity().block().getStatusCode() ));
+        Optional<ResponseEntity<Void>> block = Optional.ofNullable(clientResponse.toBodilessEntity().block());
```

Comment:
Could we have something more descriptive for the variable name e.g. `optionalClientResponse`?

Reply:
Done